enqueue.go

```diff
@@ -182,11 +182,11 @@ func (e *Enqueuer) addToKnownJobs(conn redis.Conn, jobName string) error {
 type enqueueFnType func(*int64) (string, error)
 
 func (e *Enqueuer) uniqueJobHelper(jobName string, args map[string]interface{}, keyMap map[string]interface{}) (enqueueFnType, *Job, error) {
-	useDefaultKeys := false
-	if keyMap == nil {
-		useDefaultKeys = true
-		keyMap = args
-	}
+	// useDefaultKeys := false
```

Comment:
@leguminosa the changes will cause go-craft work can't handle unique in by key.
I'll tell u the bug ya

Reply:
ok